### PR TITLE
Search for and use person in workflow step edit

### PR DESCRIPTION
**Pull request by Andreas-Tonnessen**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: [AB#232117](https://dev.azure.com/Equinor/fa63ceea-8883-41e2-8823-a3b54e4be178/_workitems/edit/232117)

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [x] I have linked my DevOps task using the AB# tag.
-   [x] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/ReleaseControl/components/Form/WorkflowEditor/WorkflowCustomEditor.tsx [hunk 1]
```diff
@@ -58,7 +58,8 @@ export const WorkflowCustomEditor = (): JSX.Element => {
             >
                 {dragableSteps.map((dragItem) => (
                     <WorkflowStep
-                        key={dragItem.item.order}
+                        //TODO - find unique ID
+                        key={Math.random()}
```
Comment (jaysencpp) [inline src/apps/ReleaseControl/components/Form/WorkflowEditor/WorkflowCustomEditor.tsx:62]: var det her

```
  {dragableSteps.map((dragItem, index) => (
                    <WorkflowStep
                        //TODO - find unique ID
                        key={index}
```

ikke fungerte?

Comment (Andreas-Tonnessen) [reply src/apps/ReleaseControl/components/Form/WorkflowEditor/WorkflowCustomEditor.tsx:62]: Ja, det fungerte ikke med index eller noen verdier som dragItem hadde som key.

File: src/apps/ReleaseControl/components/Form/WorkflowEditor/WorkflowStep.tsx [hunk 2]
```diff
@@ -55,37 +61,42 @@ export const WorkflowStep = ({ step, steps, functionalRoles }: WorkflowStepProps
...
-                            }
-                        />
+                        <StepSelect>
+                            <SingleSelect
+                                items={Object.values(ReleaseControlStepNames)}
+                                label="Step"
+                                size={30}
+                                selectedOption={step.name}
+                                readOnly={true}
+                                handleSelectedItemChange={(change) =>
```
Comment (jaysencpp) [inline src/apps/ReleaseControl/components/Form/WorkflowEditor/WorkflowStep.tsx:71]: vurder å lag egne funksjoner for handleSelectedItemChange og onSelect. Litt mer clean og lettere å lese JSX hvis det er f.eks

```
 <SingleSelect
                                items={Object.values(ReleaseControlStepNames)}
                                label="Step"
                                size={30}
                                selectedOption={step.name}
                                readOnly={true}
                                handleSelectedItemChange={handleStepSelectChange}

/>
```

**Review by jaysencpp — APPROVED** [review]

File: src/apps/ScopeChangeRequest/Components/PersonRoleSearch/PCSPersonRoleSearch.tsx [hunk 3]
```diff
@@ -78,9 +94,10 @@ export const PCSPersonRoleSearch = ({
                 }}
             >
                 <AsyncSelect
+                    defaultInputValue={value !== '' && value !== undefined ? value : undefined}
```
[the remark this answers is not in the document]
Comment (Andreas-Tonnessen) [reply src/apps/ScopeChangeRequest/Components/PersonRoleSearch/PCSPersonRoleSearch.tsx:97]: Nei, det blir ikke brukt i scope change, men det skal ikke påvirke funksjonaliteten der.

**Pull request merged**